static/sass/components/_author.scss

```diff
@@ -0,0 +1,108 @@
+// Author Box
+// ==========
+
+[data-module] {
+  + [data-module='rst-author'] {
+    border-top: 0;
+    margin-top: size('author');
+    padding-top: 0;
+  }
+}
+
+.author-summary {
+  max-width: 100%;
+}
+
+// crop triangle out of svg for birds page
+.bird-face {
+  float: left;
+  height: size('author');
+  margin-right: size('half-shim');
+  overflow: hidden;
+}
+
+.h-card {
+  [data-module='rst-author'] & {
+    margin-left: auto;
+    margin-right: auto;
+    max-width: size('full-content');
+    padding: 0;
+
+    @include above('full-content') {
+      padding: 0 size('gutter');
+      transform: translateX(negative('half-author'));
+    }
+  }
+}
+
+.bio {
+  clear: both;
+}
+
+.bio-short {
+  padding-left: size('half-author');
+  padding-top: size('pixel'); // hack to get margin to to work
+}
+
+[data-size='face'] {
+  color: color('text');
+  float: left;
+  height: size('author-h');
+  width: size('author');
+
+  [data-module='rst-author'] & {
+    shape-outside: polygon(0 0, size('author') 0, size('half-author') size('half-author'));
+    transform: translateY(-65%);
+  }
+}
+
+.socials {
+  // override main ul spacing
+  [data-region='main'] & {
+    margin: 0 0 0 size('half-author');
+    padding-bottom: size('shim');
+    padding-left: 0;
+  }
+
+  li {
+    display: inline-block;
+    list-style: none;
+  }
+
+  a {
+    background-image: none;
+    margin-right: size('shim');
+
+    [data-icon] {
+      padding: size('pixel'); // avoid social icons getting cropped
+    }
+
+
+    // Color treatments for inverted
+    .inverted & {
+      &,
+      #{$link} {
+        .icon-text {
+          @include text-underline('background', 'invert');
+          color: color('background');
+        }
+      }
+
+      #{$focus} {
+        color: color('invert-link-hover');
+
+        .icon-text {
+          background-image: linear-gradient(color('invert-link-hover'), color('invert-link-hover'));
```

Comment:
I think this is changing the color of the link underline? Since the original setting is hidden inside a mixin pattern, I think the override should be handled the same way — so it's readably clear here what we're doing. Maybe add a `text-underline-color` mixin, used by `text-underline`, but able to be called on it's own.